src/i18n/it/README.md

```diff
@@ -0,0 +1,56 @@
+# 📖 Glossario
+
+Il glossario è una raccolta di vocaboli, modi di dire e pratiche comuni per la traduzione italiana della documentazione di Astro.
+
+Alcune parole legate ad Astro e allo sviluppo web non vanno tradotte in quanto sono parte integrante del suo ambiente e non hanno una traduzione immediata nell'Italiano.
+
+## 🔄️ Vocaboli che non hanno bisogno di traduzione
+
+> 💡 Queste parole sono considerate nomi propri data la loro origine Inglese :
+
+| Vocabolo          | Definizione | Uso         |
+|-------------------|-------------|-------------|
+| Runtime           | Ambiente d'esecuzione / Tempo di esecuzione | Usato come "runtime di JavaScript" o "runtime edge" |
+| Framework         | Parola comunemente usata nel codice per specificare un software o un pacchetto che fa uso di una sintassi specifica al suo ambiente (es: `React`, `Vue`, `Svelte`, ecc...) | Utilizzato in riferimento a vari ambienti che Astro integra e supporta
+| Build             | Nel contesto di un linguaggio di programmazione che richiede un passaggio di compilazione e/o traduzione (tra due diversi linguaggi) | Questa parola deve essere usata quando si parla della fase di compilazione tramite il comando `astro build` usato per produrre un risultato compatibile con i browser e ottimizzato da Astro
+| Frontend          | Tutto quell'insieme di pacchetti e programmazione usati per costruire l'interfaccia utente direttamente visibile nel browser | Comunemente usato per distinguere il codice relativo alla logica dell'interfaccia utente dal codice che viene eseguito sul server e non è visibile all'utente |
+| Backend           | L'insieme del codice che viene usato per definire la struttura e il funzionamento del server | Vocabolo usato in contrapposizione a "Frontend" per distinguere codice e logica scritti per il browser da quelli scritti per il server |
```

Comment:
I would put the same amount of emphasis that you put in describing the Frontend. Remember that Astro is **also** a backend framework.